drizzlepac/haputils/product.py

```diff
@@ -1165,11 +1166,12 @@ def wcs_drizzle_product(self, meta_wcs):
                                   output=self.drizzle_filename,
                                   **drizzle_pars)
 
-        # Update product with SVM-specific keywords based on the footprint
+        # Update product with MVM-specific keywords based on the footprint
         with fits.open(self.drizzle_filename, mode='update') as hdu:
             for kw in self.mask_kws:
                 hdu[("SCI", 1)].header[kw] = tuple(self.mask_kws[kw])
-
+            # Add SCELLID keyword to MVM product
+            hdu[0].header['SCELLID'] = self.cell_id
```

Comment:
Maybe I am wrong, but seems simple enough to make that 0 '"SCI", 1' (but may have to move into a loop?)

Reply:
Done, as requested, in 36282ee.